### ISSUE TITLE
Add suport for an OpenCOR URI scheme

### ISSUE THREAD

**Issue by agarny** [opencor/webapp#34]
As for the legacy OpenCOR, we want to be able to have something like `opencor://openFile//home/user/file` to open `/home/user/file`.

Might want to check:
- https://www.electronjs.org/docs/latest/api/protocol
- https://www.electronjs.org/docs/latest/tutorial/launch-app-from-url-in-another-app

Support for:
- [ ] Windows;
- [x] Linux; and
- [x] macOS.

**Comment by agarny** [comment]
I imagine that PMR is the only site that relies on the OpenCOR URI scheme. So, if anything the minimum we should support (for backward compatibility) is the `openFile` endpoint?

Legacy OpenCOR already uses the `opencor` URI scheme, so should we use a different URI scheme or is it fine to also use `opencor`? I mean, if a user has both legacy OpenCOR and Web OpenCOR then whichever version was run last would be the one used when clicking on an OpenCOR-based link. I want to think that this would be fine. @nickerso, agreed?

**Comment by nickerso** [comment]
yep, I think if things are backwards compatible it makes sense to launch the new OpenCOR when available. Assuming the old OpenCOR fails gracefully if it encounters something unexpected in the URI...

**Comment by agarny** [comment]
> yep, I think if things are backwards compatible it makes sense to launch the new OpenCOR when available. Assuming the old OpenCOR fails gracefully if it encounters something unexpected in the URI...

Yeah, if you have something like `opencor://testFile//home/user/file` then it will just open or switch to legacy OpenCOR and do nothing more since it doesn't know about the `testFile` endpoint.
